### ISSUE TITLE
Explain how to run the project with conda CLI.

### ISSUE THREAD

**Issue by rohan23chhabra** [asingh33/CNNGestureRecognizer#27]
I am facing difficulty to use the project using conda CLI. Kindly instruct on the same. Update the README for it.

**Comment by asingh33** [comment]
Sure. I will update the ReadMe.

Basically its two step process:
1) Define the Keras_Backend environment variable to either tensorflow or theano.
2) Execute the  "python trackgesture.py"

On Windows:
>set "KERAS_BACKEND=tensorflow"
>python trackgesture.py

On Mac:
>KERAS_BACKEND=tensorflow python trackgesture.py

**Comment by rohan23chhabra** [comment]
Hi Abhishek,

I needed another favor from you. I was able to run the project, but the
intended output wasn't seen. The bar graph showing which gesture is
recognized is not opening properly. I am not able to figure out the output
of the algorithm on any UI. I need your help.

Thanks and regards,
Rohan Chhabra.

**Comment by asingh33** [comment]
Hey,
So once you launch the trackgesture.py you should see 4 options.
On choosing the 1st option - 'Use pretrained model for gesture recognition
& layer visualization'. You should see 3 windows - ROI, Original & Gesture
Probability.
Now to initiate the gesture prediction you need to press 'g'.
You can change the different filter modes using 'b' or 'x' based on your
need.

Hope this helps.
Abhishek

**Comment by rohan23chhabra** [comment]
Hi Abhishek,

It's always showing 'NOTHING' all the time. Its not toggling between other
gestures as I change my gesture.

Thanks and regards,
Rohan Chhabra.

**Comment by asingh33** [comment]
Yeah, I heard that one from couple of other people as well. Thats very
weird and unexpected.

Can you answer following:
- What OS are you trying on?
- Are you using Tensorflow or Theano ?
- What is the size of ori_4015imgs_weights.hdf5
<https://github.com/asingh33/CNNGestureRecognizer/blob/master/ori_4015imgs_weights.hdf5>
file
? Is it ~150 MB ?
- Also when you are trying different gesture, you are bringing your hand
within that inner green box (ROI window).
- How is your background setup. Is it dynamic or static background? Is it
possible for you to take a snapshot and send me of your desktop with those
ROI, Original & GesturePrediction windows when the prediction mode is on.

**Comment by rohan23chhabra** [comment]
OS - Linux Ubuntu. Kernel - 4.15.0
I used KERAS_BACKEND=theano python3 trackgesture.py to run the program.
Size of image = 157.4 MB
I am bringing my hand to the ROI window.
Even I don't know if its static or dynamic..

PFA the screenshot.

Thanks and Regards,
Rohan Chhabra.

**Comment by asingh33** [comment]
Unfortunately, I have never tested the code on Linux, so difficult to say
if it will require some additional tweaking.

**Comment by rohan23chhabra** [comment]
Hi abhishek,

Things turned out pretty well on Windows as the application ran
successfully. I just wanted to know what are the accuracies of CNN model,
training accuracy and testing accuracy?

Thanks and Regards,
Rohan.
